tests/SpaceTest.php

```diff
@@ -82,7 +82,7 @@ public function testCanPublicUploadText()
     public function testCanUploadFile()
```

Comment:
Please add a second test, so there is one without a mimetype and one with

Reply:
> Please add a second test, so there is one without a mimetype and one with

Right! I was looking at the tests and some didn't pass, should I just add a new method to the test with mime-type or will I have to make other changes to the entire file?

Sorry for the late to response.

Reply:
@MacroMan Hello, see my last commit on this conversation.